### DISCUSSION TITLE
allow to adjust the php memory limit

### DISCUSSION THREAD

**Discussion by WeiChihChern** [nextcloud/all-in-one#1354] [Ideas] [answered]
As titled.  Cannot find any information regarding this.

**Comment by szaimen** [comment, marked as answer]
Will be handled in https://github.com/nextcloud/all-in-one/pull/1382

**Reply by szaimen** [reply]
This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by ghost** [reply]
@szaimen How can I increase the memory limit? My instance feels extremely slow, even though I have 4 GB of RAM (only 1 GB is being used)

**Reply by szaimen** [reply]
Hi, see https://github.com/nextcloud/all-in-one#how-to-adjust-the-php-memory-limit-for-nextcloud

**Reply by szaimen** [reply]
Regarding speed see https://github.com/nextcloud/all-in-one/discussions/1335
